test-fixture/src/lib.rs

```diff
@@ -53,7 +53,7 @@ pub fn fixture_init() {
 
 // This needs to be > 2ms to avoid it being rounded to zero.
 // NSS operates in milliseconds and halves any value it is provided.
-pub const ANTI_REPLAY_WINDOW: Duration = Duration::from_millis(10);
+pub const ANTI_REPLAY_WINDOW: Duration = Duration::from_millis(10000);
```

Comment:
I think that I would prefer if you changed the specific test to use a different anti-replay window instead.

This value is deliberately very tight so that tests fail when there are significant changes in RTT.  You can change the server to have a different window by setting the value explicitly.  I'll send you a PR.

Reply:
https://github.com/larseggert/neqo/pull/30